hypothesis-python/tests/cover/test_lookup_py38.py

```diff
@@ -32,6 +33,28 @@ def test_typing_Final(data):
     assert isinstance(value, int)
 
 
+@pytest.mark.parametrize(
+    "annotated_type,expected_strategy",
+    [
+        (typing_extensions.Annotated[int, "foo"], st.integers()),
+        (typing_extensions.Annotated[typing.List[float], "foo"], st.lists(st.floats())),
+        (
+            typing_extensions.Annotated[typing_extensions.Annotated[str, "foo"], "bar"],
+            st.text(),
+        ),
+        (
+            typing_extensions.Annotated[
+                typing_extensions.Annotated[typing.List[typing.Dict[str, bool]], "foo"],
+                "bar",
+            ],
+            st.lists(st.dictionaries(keys=st.text(), values=st.booleans())),
+        ),
+    ],
+)
+def test_typing_Annotated(annotated_type, expected_strategy):
```

Comment:
We can delete this test I think; it'll be covered by the `/typing_extensions/` test below.